backend/app/Http/Resources/QuizLogResource.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace App\Http\Resources;
+
+use Illuminate\Http\Resources\Json\JsonResource;
+
+class QuizLogResource extends JsonResource {
+    public function toArray($request) {
+        $jsonArr = [];
+        foreach ($this->jsonKeyMap as $k => $v) {
+            $jsonArr[$v] = $this->$k;
+        }
+        return $jsonArr;
```

Comment:
new line before return
```suggestion
        }
        
        return $jsonArr;
```

Reply:
done